From eb12c16e61dcb2355ae2cd7f592d5b7f0b12c0c0 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:50:14 -0400
Subject: [tools] perf/symbols: Rename [sl]name_alloc to match the members they refer to

Message-id: <1408953210-26343-127-git-send-email-jolsa@redhat.com>
Patchwork-id: 88863
O-Subject: [PATCH RHEL7.1 BZ1133083 126/322] perf symbols: Rename [sl]name_alloc to match the members they refer to
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit c7282f2efff9f115378b450b7aea51210fabb6ef
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Tue Dec 10 10:44:37 2013 -0300

description
===========
So we now have:

   dso->short_name
   dso->short_name_len
   dso->short_name_allocated

Ditto for the 'long  variants. To more quickly grasp what they refer to.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/dso.c b/tools/perf/util/dso.c
index a0c7c59..55c9835 100644
--- a/tools/perf/util/dso.c
+++ b/tools/perf/util/dso.c
@@ -465,9 +465,9 @@ void dso__delete(struct dso *dso)
  int i;
  for (i = 0; i < MAP__NR_TYPES; ++i)
   symbols__delete(&dso->symbols[i]);
- if (dso->sname_alloc)
+ if (dso->short_name_allocated)
   free((char *)dso->short_name);
- if (dso->lname_alloc)
+ if (dso->long_name_allocated)
   free(dso->long_name);
  dso_cache__free(&dso->cache);
  dso__free_a2l(dso);
diff --git a/tools/perf/util/dso.h b/tools/perf/util/dso.h
index 384f2d9..00a232d 100644
--- a/tools/perf/util/dso.h
+++ b/tools/perf/util/dso.h
@@ -89,8 +89,8 @@ struct dso {
  u8   has_srcline:1;
  u8   hit:1;
  u8   annotate_warned:1;
- u8   sname_alloc:1;
- u8   lname_alloc:1;
+ u8   short_name_allocated:1;
+ u8   long_name_allocated:1;
  u8   sorted_by_name;
  u8   loaded;
  u8   rel;
diff --git a/tools/perf/util/machine.c b/tools/perf/util/machine.c
index bac817a..f66f309 100644
--- a/tools/perf/util/machine.c
+++ b/tools/perf/util/machine.c
@@ -764,7 +764,7 @@ static int map_groups__set_modules_path_dir(struct map_groups *mg,
     goto out;
    }
    dso__set_long_name(map->dso, long_name);
-   map->dso->lname_alloc = 1;
+   map->dso->long_name_allocated = 1;
    dso__kernel_module_get_build_id(map->dso, "");
   }
  }
@@ -936,7 +936,7 @@ static int machine__process_kernel_mmap_event(struct machine *machine,
    goto out_problem;
 
   map->dso->short_name = name;
-  map->dso->sname_alloc = 1;
+  map->dso->short_name_allocated = 1;
   map->end = map->start + event->mmap.len;
  } else if (is_kernel_mmap) {
   const char *symbol_name = (event->mmap.filename +
diff --git a/tools/perf/util/symbol.c b/tools/perf/util/symbol.c
index de87dba..265a149 100644
--- a/tools/perf/util/symbol.c
+++ b/tools/perf/util/symbol.c
@@ -1458,7 +1458,7 @@ int dso__load_vmlinux_path(struct dso *dso, struct map *map,
  if (filename != NULL) {
   err = dso__load_vmlinux(dso, map, filename, filter);
   if (err > 0) {
-   dso->lname_alloc = 1;
+   dso->long_name_allocated = 1;
    goto out;
   }
   free(filename);
@@ -1468,7 +1468,7 @@ int dso__load_vmlinux_path(struct dso *dso, struct map *map,
   err = dso__load_vmlinux(dso, map, vmlinux_path[i], filter);
   if (err > 0) {
    dso__set_long_name(dso, strdup(vmlinux_path[i]));
-   dso->lname_alloc = 1;
+   dso->long_name_allocated = 1;
    break;
   }
  }
@@ -1612,7 +1612,7 @@ static int dso__load_kernel_sym(struct dso *dso, struct map *map,
   if (err > 0) {
    dso__set_long_name(dso,
         strdup(symbol_conf.vmlinux_name));
-   dso->lname_alloc = 1;
+   dso->long_name_allocated = 1;
    return err;
   }
   return err;
-- 
1.7.1